package.json

```diff
@@ -32,10 +32,10 @@
   "dependencies": {
     "body": "^5.0.0",
     "error": "^5.0.0",
-    "farmhash": "^0.2.0",
+    "farmhash": "^1.1.0",
     "metrics": "^0.1.8",
     "node-uuid": "^1.4.3",
-    "toobusy": "^0.2.4",
+    "toobusy-js": "^0.5.0",
```

Comment:
You need to find the place where `toobusy` is required, and replace it with `require('toobusy-js')`